Dockerfile

```
@@ -0,0 +1,6 @@
+FROM openjdk:17-jdk AS builder
+WORKDIR /app
+COPY --from=builder /app/build/libs/*.jar app.jar
```

Hadolint (2.12.0)

[error] `COPY --from` cannot reference its own `FROM` alias

(DL3023)